operator/pkg/secretsync/secretsync_reconcile_test.go

```diff
@@ -194,7 +196,7 @@ var secretFixture = []client.Object{
 }
 
 func Test_SecretSync_Reconcile(t *testing.T) {
-	logger := hivetest.Logger(t)
+	logger := hivetest.Logger(t, hivetest.LogLevel(slog.LevelDebug))
```

Comment:
is this changed for testing purposes or here to stay?

Reply:
All of the logs are debug-level, so if a test fails, you get no logs without this. So, here to stay, imho.